### ISSUE TITLE
Compatibility with Chisels & Bits?

### ISSUE THREAD

**Issue by uecasm** [rwtema/FunkyLocomotion#102]
Chisels & Bits seems to be the new main multipart mod for 1.10, but its saw doesn't work on frame blocks, leaving no apparent way to create multipart frame blocks in this MC version.  (If there's another mod that adds a Funky Locomotion-compatible saw, I can't find it.)

For what I'm trying to do right now, all I really want is the ability to retexture one side of the frame block (similar to #97), but full multipart compatibility would permit cabling to run through frames, which can be handy as well.

**Comment by Revvilo** [comment]
Yeah, this would be great to have again, muddled up my build for a frame miner due to not being able to get the items from the block breakers at the front (without laborious crafting of wireless block access).

**Comment by Recer75** [comment]
I am also having the same problem as Revvilo
